crates/ruff_python_formatter/src/comments/format.rs

```diff
@@ -227,6 +227,46 @@ impl Format<PyFormatContext<'_>> for FormatDanglingComments<'_> {
     }
 }
 
+/// Formats the dangling comments within a parenthetical, for example:
```

Comment:
Nit: I find `parenthetical` difficult to understand. I think we just used the term `parenthesized` for terms enclosed by any parentheses.